File: 06-code-golf/tf-06.py
#!/usr/bin/env python
import heapq, re, sys

words = re.findall("[a-z]{2,}", open('../pride-and-prejudice.txt').read().lower())
for w in heapq.nlargest(25, set(words) - set(open("../stop_words.txt").read().split(",")), words.count):
    print (w, "-", words.count(w))

File: 06-code-golf/tf-06.cs
using System;
using System.IO;
using System.Linq;
using System.Text.RegularExpressions;

namespace _06_code_golf
{
    class Program
    {
        static void Main(string[] args)
        {
            var stops = File.ReadAllText("../stop_words.txt").Split(',').Union(Enumerable.Range('a', 26).Select(c => Convert.ToChar(c).ToString()));
		
		    var words = from x in Regex.Split(File.ReadAllText("../pride-and-prejudice.txt"), "[^a-zA-Z]+")
			where !stops.Contains(x.ToLower()) && !string.IsNullOrWhiteSpace(x)
			select x.ToLower();
		
            var uniqu_words =  words.GroupBy(w => w).OrderByDescending(w => w.Count());
            
            Console.WriteLine(string.Join("\n", uniqu_words.Take(25).Select(x => $"{x.Key}  -  {words.Count(w => w == x.Key)}")));
        }
    }
}
